napalm_ios/ios.py

```diff
@@ -27,6 +27,7 @@
 from napalm_base.exceptions import ReplaceConfigException, MergeConfigException
 from napalm_base.utils import py23_compat
 import napalm_base.constants as C
+import napalm_base.helpers
```

Comment:
I think `mac` is a better name too. Readable code is nice : )  If we need the error handling we can decorate the method or call a common function to do it from within `mac` and friends.